catalogue_api/api/src/main/scala/uk/ac/wellcome/platform/api/services/WorksService.scala

```diff
@@ -1,19 +1,24 @@
 package uk.ac.wellcome.platform.api.services
 
 import com.google.inject.{Inject, Singleton}
-import com.sksamuel.elastic4s.http.search.SearchHit
+import com.sksamuel.elastic4s.http.search.{SearchHit, SearchResponse}
 import io.circe.Decoder
 import uk.ac.wellcome.json.JsonUtil._
 import uk.ac.wellcome.models.work.internal.{IdentifiedBaseWork, IdentifiedWork}
-import uk.ac.wellcome.platform.api.models.{ApiConfig, ResultList}
+import uk.ac.wellcome.platform.api.models.ResultList
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success}
 
+case class WorksSearchOptions(
+  workTypeFilter: Option[String],
+  indexName: String,
+  pageSize: Int,
+  pageNumber: Int
+)
+
 @Singleton
-class WorksService @Inject()(
-  apiConfig: ApiConfig,
```

Comment:
One small nicety: we get to ditch the dependency between WorksService and ApiConfig, which was always a bit odd – the default page size now gets passed higher up, in the WorksController.